app/models/loader/types.rb

```diff
@@ -170,17 +185,34 @@ def identifier
     class Group < Record
       def_delegators :@policy_object, :gidnumber
 
+      def verify; end
+
       def create!
         self.annotations ||= {}
         self.annotations["conjur/gidnumber"] ||= self.gidnumber if self.gidnumber
 
         super
       end
     end
-    
+
     class User < Record
       def_delegators :@policy_object, :public_keys, :account, :role_kind, :uidnumber, :restricted_to
 
+      # Below is a sample method verifying policy data validity
+      def verify
```

Comment:
Why is `calculate_defaults!` completely commented out but method declaration of `verify` is not commented?
Just making sure you're not missing anything by mistake.

Reply:
because I dont mandate calculate_defaults implementation, but do verify! implementation.
There can be entity without defaults. entity without verify will through exception, and thus verify method must exist